contents/tutorials/when-and-how-to-run-group-targeted-a-b-tests.md

```diff
@@ -0,0 +1,88 @@
+
+A B2B SaaS company wants to see if a new pricing plan helps retain customers. They split companies into two groups: one with the new plan and one with the old plan. However, the randomization process assigns more small companies to the new plan and more large companies to the old plan. This may skew the results because company size could be a factor in determining customer retention rather than the pricing plan itself.
+
+### 3. Fewer user-level insights
+
+Since group-targeted experiments provide insights on an aggregate level, they may not show insights on a user level.
+
+For example, say a B2B SaaS tool wants users to invite coworkers to use the app. They add a big "Invite Coworker" button and test it in a company-targeted experiment. The results show more invites are sent and new users join, but it's hard to know how individual users used the button. For example, what are the roles or skill sets of team members who are most likely to interact with the button and send invites?
+
+## Combining user-targeted and group-targeted experiments to reduce risk
+
+To improve the quality of insights from group-targeted experiments and address the risks mentioned above, you may want to consider running an experiment with your change at a user-level before running it at a group-level. This can help you in the following ways:
+
+### 1. Save time for changes that don't have an impact. 
+
+As mentioned, lower statistical power in group-targeted experiments means you usually need to run your experiments longer. However, a change that does not impact behavior at a user-level will not translate into impact at a group-level. So you may waste time waiting for results from a group-level test that has no impact.
+
+In this case, it may be worth running a user-targeted experiment first in order to see if there is any impact on individual user behavior (which should be quicker to gain insights from because of higher statistical power) before running a group-targeted experiment.
+
+
+### 2. Gain user-level insights and identify randomization risks
+By running a user-targeted experiment first, you'll gain insight into how individual users interact with your change and what characteristics they may possess. This will then give you an understanding of what results you can expect from a group-targeted experiment.
+
+This can also help you identify randomization risks. For example, let's say a B2B SaaS app runs a user-targeted experiment and notices that users in large companies are more likely to interact with their new feature. In this case, when they run a group-targeted experiment, they can ensure to filter out small and medium-sized companies in their experiment in order to obtain more accurate results.
+
+
+## How to setup group-targeted experiments on PostHog
```

Comment:
If this is a tutorial, then move this up.

If this is going to be a blog, then you can just remove this and rename the piece "when and why to run group targeted a/b tests" (or something like that.